### ISSUE TITLE
Wish help: Does the tf.Session.run() function process each tensor one-by-one in a batch??

### ISSUE THREAD

**Issue by Dee-Ma** [tensorflow/tensorflow#25082]
Hi,

Currently, I would like to use the tf.Session.run() function to train a CNN model. For this mission, the input data to the model is a batch (4D, [batch_size, height of image, width of image, depths]) contains 16 image blocks. My understanding is that the 16 blocks should be processed by the CNN model one-by-one, and then calculate the mean loss based on the 16 outputs to update the model parameters (one batch data has been processed).  

May I ask if the tf.session.run() will process the image block (a tensor) one-by-one?

Thank you in advance.

**Comment by jvishnuvardhan** [comment]
Please post this kind of support questions at [Stackoverflow](https://stackoverflow.com/questions/tagged/tensorflow). There is a big community to support and learn from your questions. Github is mainly for addressing bugs in installation and performance. Thanks!
